### DISCUSSION TITLE
Add native sponsors option in app

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#43224] [Mobile]
Please add a sponsor option in the app when browsing repositories.

Even better if it used the in-app purchasing system from Android/iOS. Even if it would net me 30% less income.

If it means the difference between people not sponsoring and sponsoring, it's better than nothing.

**Comment by HenSquared** [comment]
Hi thanks for the feature request. This is an exciting idea. I've opened up an internal issue to track and discuss with the team. Thanks!

**Comment by andreynering** [comment]
I suppose the 30% cut is a huge issue that would prevent this to become reality.
